package-lock.json

```diff
@@ -1,8 +1,2751 @@
 {
-  "name": "webrtc-hello-world",
+  "name": "webrtc-voicebridge",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
looks good to me, this lock file version seems to indicate usually you're going from LTS (v14) to a non-LTS more modern version. Either is fine, but someone not on this version could potentially have build issues

Reply:
Interesting! Not terribly familiar with Node but I had thought that was only relevant to NPM (and worked with v6 or v7).

Reply:
You are correct! Just read up, seems backwards compatible. v3 is where things get tricky, all is good!

https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json#lockfileversion

Reply:
Ha, I wasn't even aware that there was a v3.